connector/doc/guides/bootstrap_connector.rst

```diff
@@ -208,6 +211,23 @@ created. I propose to create a helper too in
         return checkpoint.add_checkpoint(session, model_name, record_id,
                                          'coffee.backend', backend_id)
 
+Message checkpoint
+------------------
+
+When you need to show a plain text message to a user
+(for instance: an import job did not succeeded),
```

Comment:
Indeed, the issue we were thinking about that led us to this approach was: "I want to display a nice message to the end user instead of an error traceback, whenever I don't have an object".

Imagine you are importing a product and the product is not in odoo yet, and for some reason it fails (maybe because you put an hard constraint in the import chain). At this point you do not have any object to attach the checkpoint to and the only way to see this is to go and search for failed jobs.

This could be the good example, I guess :)